deepr/jobs/export_xla_model_metadata.py

```diff
@@ -0,0 +1,100 @@
+"""Export xla compatible model metadata from a saved model"""
+
+from dataclasses import dataclass
+import logging
+from typing import List, Dict
+
+import tensorflow as tf
+
+from deepr.io import Path
+from deepr.jobs import base
+from deepr.utils import import_graph_def
+from deepr.jobs._xla_generated.tf2xla_pb2 import Config as XlaMetaConfig
```

Comment:
It's unfortunate that this is not importable from standard tf.
An alternative to prefixing the module by `_` is to move it to utils or something like this

Reply:
moved